misc/drone-scripts/setup.sh

```diff
@@ -19,22 +19,29 @@ govc snapshot.revert -vm $ESX_6_0 init
 
 echo "Waiting for revert to complete";
 
+DIR=$(dirname ${BASH_SOURCE[0]})
+. $DIR/../scripts/wait_for.sh
+
+# Threshold to time out 
+retryCount=30
+
 echo ESX 6.5
-until govc vm.ip $ESX_6_5
-do
-    echo "Waiting for revert to complete for ESX6.5";
-    sleep 1;
-done
+wait_for "govc vm.ip $ESX_6_5" $retryCount
 
 echo ESX 6.0
-until govc vm.ip $ESX_6_0
-do
-    echo "Waiting for revert to complete for ESX6.0";
-    sleep 1;
-done
+wait_for "govc vm.ip $ESX_6_0" $retryCount
 
 echo "Reset complete"
-sleep 5;
+
+export GOVC_URL=$GOVC_URL_6_5
+export GOVC_USERNAME=$GOVC_USERNAME_ESX
+export GOVC_PASSWORD=$GOVC_PASSWORD_ESX
+
```

Comment:
Do we need to set GOVC_DATACENTER also? The VMs will be configured in one in VC.